README.md

```diff
@@ -79,9 +79,12 @@ By default, initial provisioning and setup tasks are configured to be skipped.
 To run playbook without skipping setup tasks, override the default skip tag configuration:
 
 ```{bash}
-ansible-playbook playbooks/name_of_playbook.yml --skip-tags none
+env ANSIBLE_SKIP_TAGS= ansible-playbook playbooks/name_of_playbook.yml
 ```
 
+Note that `--skip-tags=[]` doesn't work because the skip tags setting in
```

Comment:
Nice, glad we added a note about this, I remember it confusing us yesterday

Reply:
Yeah, I had to read up on the proper syntax and figured out the order of precedence so I could come up with something that works. Makes me wonder if we're setting the default in the wrong place, but this will work for now.